openapi/services/journals.yaml

```diff
@@ -0,0 +1,129 @@
+Summary:
+  get:
+    summary: Summary information about a publication indexed by ADS
+    parameters:
+      - $ref: '../parameters.yaml#/JournalBibstem'
+    description: >
+      For a bibstem, return information about the publication, including the
+      official name and commonly-used variations, publisher and publisher
+      history (when available), and the ISSN and other identifiers when
+      available.
+    tags:
+      - journals
+    security:
+      - ApiKeySecurity: []
+    responses:
+      '200':
+        description: >
+          JSON object containing `master`, `idents`, `abbrev`, `pubhist`, and
+          `names` objects, which contain summary information about the
+          publication (master), known alphanumeric identifiers (idents),
+          known abbreviations for the title (abbrev), historical information
+          about the journal and its publisher(s) when available (pubhist),
+          and native-language and transliterated names for journals having
+          names in languages other than English, when available (names).
+        content:
+          application/json:
+            schema:
+              allOf:
+                - $ref: '../schemas.yaml#/summary'
+      '403':
+        description: Authentication failure, unable to get results
+      '500':
+        description: Internal error, unable to get results
+Journal:
+  get:
+    summary: >
+      Indexed journals that match a user-supplied search string
+    description: >
```

Comment:
You can fill an example in the param (`openapi/parameters.yaml`), which might be useful for this one, since you're providing examples anyway. See https://github.com/adsabs/adsabs-dev-api/blob/master/openapi/parameters.yaml#L152 for an example